packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-card/gux-selector-card.tsx

```diff
@@ -100,7 +100,6 @@ export class GuxSelectorCard {
           [`gux-${this.variant}`]: true,
           'gux-disabled': this.disabled
         }}
-        onClick={() => this.input.setAttribute('checked', 'true')}
```

Comment:
I removed this because I think it wasn't having an impact except when the description was clicked. I looked at applying a click handler to the description solely. However, this causes some issues. 

The checked attribute can't be relied on to set the state of the checkbox programmatically. It causes issues when the user interacts with the component. 

Triggering a click on the input when the description is clicked will set the checkbox to the correct state. However, the element will not receive focus which is the expected behaviour on Chrome and Firefox. While it's possible to just add some focus programmatically also, this doesn't match with the expected input behaviour on Safari. 

At this point, I think it's probably a case of either have the tooltip and clicking the description does nothing OR no tooltip and the description truncates but clicking on it selects the card. What do you think @gavin-everett-genesys @daragh-king-genesys ?

Reply:
Yeah I think the `onClick` should be removed . I could not find any differences between how the `label` is setup and how the `description` is setup and the label seems to be working fine. I think it must be possible to have that when you click the description it will focus and also have the ability to show the tooltip. It may be a `zindex` issue. I am not 100% sure.